cmd/livepeer/livepeer.go

```diff
@@ -329,6 +339,12 @@ func main() {
 	}
 }
 
+func CheckTranscoderAvailability(orch Orchestrator) {
```

Comment:
Where is this func being used? - wasn't able to find any reference to it

Reply:
This is a great problem to have ... usually it's the opposite - no one around to do PR reviews!

Reply:
WIP labels are a great idea 👍